2023-04-08 14:47:59,600 INFO    MainThread:3630 [wandb_setup.py:_flush():68] Configure stats pid to 3630
2023-04-08 14:47:59,600 INFO    MainThread:3630 [wandb_setup.py:_flush():68] Loading settings from /home/paperspace/.config/wandb/settings
2023-04-08 14:47:59,600 INFO    MainThread:3630 [wandb_setup.py:_flush():68] Loading settings from /home/paperspace/home/pystack/notebooks/wandb/settings
2023-04-08 14:47:59,600 INFO    MainThread:3630 [wandb_setup.py:_flush():68] Loading settings from environment variables: {'_require_service': 'True'}
2023-04-08 14:47:59,600 INFO    MainThread:3630 [wandb_setup.py:_flush():68] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2023-04-08 14:47:59,600 INFO    MainThread:3630 [wandb_init.py:_log_setup():476] Logging user logs to /home/paperspace/home/pystack/notebooks/wandb/run-20230408_144759-rlidpfb9/logs/debug.log
2023-04-08 14:47:59,600 INFO    MainThread:3630 [wandb_init.py:_log_setup():477] Logging internal logs to /home/paperspace/home/pystack/notebooks/wandb/run-20230408_144759-rlidpfb9/logs/debug-internal.log
2023-04-08 14:47:59,600 INFO    MainThread:3630 [wandb_init.py:_jupyter_setup():426] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f774b172670>
2023-04-08 14:47:59,601 INFO    MainThread:3630 [wandb_init.py:init():516] calling init triggers
2023-04-08 14:47:59,601 INFO    MainThread:3630 [wandb_init.py:init():519] wandb.init called with sweep_config: {}
config: {}
2023-04-08 14:47:59,601 INFO    MainThread:3630 [wandb_init.py:init():535] re-initializing run, found existing run on stack: 1jjxfrsa
2023-04-08 14:47:59,602 INFO    MainThread:3630 [wandb_run.py:_finish():1746] finishing run ds-ml/pystack/1jjxfrsa
2023-04-08 14:47:59,602 INFO    MainThread:3630 [wandb_run.py:_atexit_cleanup():1949] got exitcode: 0
2023-04-08 14:47:59,602 INFO    MainThread:3630 [wandb_run.py:_restore():1932] restore
2023-04-08 14:47:59,603 INFO    MainThread:3630 [wandb_run.py:_restore():1938] restore done
2023-04-08 14:48:03,632 INFO    MainThread:3630 [wandb_run.py:_footer_history_summary_info():3377] rendering history
2023-04-08 14:48:03,633 INFO    MainThread:3630 [wandb_run.py:_footer_history_summary_info():3409] rendering summary
2023-04-08 14:48:03,634 INFO    MainThread:3630 [wandb_run.py:_footer_sync_info():3333] logging synced files
2023-04-08 14:48:03,638 INFO    MainThread:3630 [wandb_init.py:init():569] starting backend
2023-04-08 14:48:03,638 INFO    MainThread:3630 [wandb_init.py:init():573] setting up manager
2023-04-08 14:48:03,640 INFO    MainThread:3630 [backend.py:_multiprocessing_setup():102] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2023-04-08 14:48:03,641 INFO    MainThread:3630 [wandb_init.py:init():580] backend started and connected
2023-04-08 14:48:03,647 INFO    MainThread:3630 [wandb_run.py:_label_probe_notebook():1113] probe notebook
2023-04-08 14:48:03,647 INFO    MainThread:3630 [wandb_run.py:_label_probe_notebook():1123] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2023-04-08 14:48:03,647 INFO    MainThread:3630 [wandb_init.py:init():658] updated telemetry
2023-04-08 14:48:03,657 INFO    MainThread:3630 [wandb_init.py:init():693] communicating run to backend with 60 second timeout
2023-04-08 14:48:04,827 INFO    MainThread:3630 [wandb_run.py:_on_init():2000] communicating current version
2023-04-08 14:48:04,869 INFO    MainThread:3630 [wandb_run.py:_on_init():2004] got version response upgrade_message: "wandb version 0.14.2 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2023-04-08 14:48:04,870 INFO    MainThread:3630 [wandb_init.py:init():728] starting run threads in backend
2023-04-08 14:48:04,901 INFO    MainThread:3630 [wandb_run.py:_console_start():1980] atexit reg
2023-04-08 14:48:04,901 INFO    MainThread:3630 [wandb_run.py:_redirect():1838] redirect: SettingsConsole.WRAP_RAW
2023-04-08 14:48:04,901 INFO    MainThread:3630 [wandb_run.py:_redirect():1903] Wrapping output streams.
2023-04-08 14:48:04,901 INFO    MainThread:3630 [wandb_run.py:_redirect():1925] Redirects installed.
2023-04-08 14:48:04,902 INFO    MainThread:3630 [wandb_init.py:init():765] run started, returning control to user process
2023-04-08 14:49:21,125 INFO    MainThread:3630 [wandb_init.py:_pause_backend():389] pausing backend
2023-04-08 14:49:21,126 INFO    MainThread:3630 [jupyter.py:save_ipynb():378] not saving jupyter notebook
2023-04-08 14:53:17,935 INFO    MainThread:3630 [wandb_init.py:_resume_backend():398] resuming backend
2023-04-08 14:53:17,937 INFO    MainThread:3630 [wandb_run.py:_finish():1746] finishing run ds-ml/pystack/rlidpfb9
2023-04-08 14:53:17,937 INFO    MainThread:3630 [jupyter.py:save_history():448] not saving jupyter history
2023-04-08 14:53:17,937 INFO    MainThread:3630 [jupyter.py:save_ipynb():378] not saving jupyter notebook
2023-04-08 14:53:17,937 INFO    MainThread:3630 [wandb_init.py:_jupyter_teardown():408] cleaning up jupyter logic
2023-04-08 14:53:17,937 INFO    MainThread:3630 [wandb_run.py:_atexit_cleanup():1949] got exitcode: 0
2023-04-08 14:53:17,938 INFO    MainThread:3630 [wandb_run.py:_restore():1932] restore
2023-04-08 14:53:17,938 INFO    MainThread:3630 [wandb_run.py:_restore():1938] restore done
